Assets/Scripts/Models/Character.cs

```diff
@@ -329,7 +338,7 @@ private bool CheckForJobMaterials()
                 // Walk towards a tile containing the required goods.
                 Debug.Log("Walk to the stuff");
                 Debug.Log(myJob.canTakeFromStockpile);
-
+                
```

Comment:
Theres a handful of lines in this file with trailing white space. Could you fix that? :)